bindings/core/src/method/secret_manager.rs

```diff
@@ -36,40 +37,40 @@ pub enum SecretManagerMethod {
     SignatureUnlock {
         /// Transaction signing hash
         transaction_signing_hash: String,
-        /// Chain used to sign the hash
-        #[serde(with = "Bip44Def")]
-        chain: Bip44,
+        /// Options used to sign the hash
+        signing_options: serde_json::Value,
```

Comment:
Should all the signing options be optional? For something like a PrivateKeySecretManager that can't have multiple keys one otherwise needs to provide something empty just so this field is not missing?

Reply:
I'm going to make them default to `Value::Null`, which will deserialize to `()` correctly (I double checked).